lambda/otel/layer/Gemfile

```diff
@@ -2,10 +2,8 @@
 
 source 'https://rubygems.org'
 
-gem 'opentelemetry-instrumentation-aws_lambda', '~> 0.1.0'
-
 source 'https://rubygems.pkg.github.com/xuan-cao-swi' do
-  gem 'solarwinds_apm', '6.0.0.prev7'
+  gem 'solarwinds_apm', '6.0.6'
```

Comment:
What's the reason for a custom version of the library?

Reply:
It's all temporary. I will make a release for lambda support and replace the version and repo url point to official rubygems.org

Reply:
Is it possible to make a release so we stop using a custom build of the custom distro? Quick scan of the changes https://github.com/solarwinds/apm-ruby/compare/6.0.0...main they shouldn't impact the non-lambda custom distro and it would be a good checkpoint to go through the release, confirming expected behaviour for non-lambda, and allow this PR and any follow-on lambda work to be based on more solid foundations.

Reply:
Updated:
[ARM](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/layers/solarwinds-apm-ruby-arm64-6_0_1/versions/1?tab=versions)
[AMD](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/layers/solarwinds-apm-ruby-x86_64-6_0_1/versions/1?tab=versions)